homeassistant/components/simplisafe/__init__.py

```diff
@@ -113,8 +113,9 @@ def _async_save_refresh_token(hass, config_entry, token):
             try:
                 await system.update()
```

Comment:
If order is not important we could use:
```py
tasks = [system.update() for system in system]
results = await asyncio.gather(*tasks, return_exceptions=True)
```
And then iterate the results and check with `isinstance` for the exception, and if so log error and continue, as below.

Reply:
Can you help me understand the benefits?

Reply:
sequential vs parallel execution.

Reply:
Now with image
![image](https://user-images.githubusercontent.com/1444314/55127242-be6ee580-50cc-11e9-9fc5-a733c2bf30dd.png)

Reply:
Afterwards, you can just iterate over the results:

```python
for system, result in zip(systems, results):
    …
```